Platform/OpenCanopy/OpenCanopy.c

```diff
@@ -1356,12 +1493,11 @@ GuiGetInterpolatedValue (
   UINT32 DeltaTime;
 
   ASSERT (Interpol != NULL);
-  ASSERT (Interpol->StartTime <= CurrentTime);
```

Comment:
Fritz, is this ok to remove this?

Reply:
It's to support sequential animations: one animation starts after the end of the previous one, so StartTime can be less than CurrentTime.